games-roguelike/cataclysm-dda/cataclysm-dda-0h.ebuild

```diff
@@ -0,0 +1,183 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg-utils toolchain-funcs flag-o-matic
+
+SP_VER="2024-10-27"
+
+DESCRIPTION="A turn-based survival game set in a post-apocalyptic world"
+HOMEPAGE="https://cataclysmdda.org
+	https://github.com/CleverRaven/Cataclysm-DDA"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/CleverRaven/Cataclysm-DDA.git"
+	SLOT="${PV}"
+else
+	v2="$(ver_cut 2)"
+	MY_PV="$(ver_cut 1).${v2^^}"
+	unset v2
+	SRC_URI="https://github.com/CleverRaven/Cataclysm-DDA/archive/${MY_PV}-RELEASE.tar.gz -> ${P}.tar.gz \
+		soundpack? ( https://github.com/Fris0uman/CDDA-Soundpacks/releases/download/${SP_VER}/CC-Sounds.zip -> ${P}-soundpack.zip )"
+	SLOT="${MY_PV}"
+	S="${WORKDIR}/Cataclysm-DDA-${MY_PV}-RELEASE"
+	KEYWORDS="~amd64"
+fi
+
+# CC-BY-SA-4.0 for soundpack
+LICENSE="Apache-2.0 CC-BY-SA-3.0 CC-BY-SA-4.0 MIT OFL-1.1 Unicode-3.0"
+IUSE="debug ncurses nls +sound +soundpack test +tiles"
+REQUIRED_USE="soundpack? ( sound ) sound? ( tiles ) \
+	|| ( tiles ncurses )"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	ncurses? ( sys-libs/ncurses )
+	tiles? (
+		media-libs/libsdl2[video]
+		media-libs/sdl2-image[png]
+		media-libs/sdl2-ttf[harfbuzz]
+		sound? (
+			   media-libs/libsdl2[sound]
+			   media-libs/sdl2-mixer[vorbis]
+		)
+	)"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	soundpack? ( app-arch/unzip )
+	nls? ( sys-devel/gettext )
+	"
+
+src_unpack() {
+	if [[ ${PV} == 9999 ]]; then
+		git-r3_src_unpack
+
+		if use soundpack; then
+			git-r3_fetch https://github.com/Fris0uman/CDDA-Soundpacks
+			git-r3_checkout https://github.com/Fris0uman/CDDA-Soundpacks "${WORKDIR}/CDDA-Soundpacks"
```

Comment:
Can you move `"${WORKDIR}/CDDA-Soundpacks/sound/CC-Sounds` to `${WORKDIR}/CC-Sounds` so we can reduce the if condition in `src_install`?